<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="ProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>270</width>
     <height>15</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="boxFileHandling">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>145</width>
     <height>175</height>
    </rect>
   </property>
   <property name="title">
    <string>File Handling</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>141</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QPushButton" name="buttonLoad">
       <property name="text">
        <string>Load Mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save As</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>100</y>
      <width>131</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Preset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidgetFunctions">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>199</y>
     <width>300</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="TabSampling">
    <attribute name="title">
     <string>Sampling</string>
    </attribute>
    <widget class="QSlider" name="sliderSubset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>240</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>33</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>275</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Target Size of Subsampled Mesh         [in %]</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSliderValue">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>85</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>33</string>
     </property>
     <property name="textFormat">
      <enum>Qt::LogText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxAdaptivity">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>114</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>0.900000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.650000000000000</double>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>50</y>
       <width>280</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAdaptivity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>115</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Adaptivity Manualy</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonCalculateWeights">
        <property name="text">
         <string>Weights</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRunSampling">
        <property name="text">
         <string>Sample</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="TabReIndexing">
    <attribute name="title">
     <string>ReMeshing</string>
    </attribute>
    <widget class="QPushButton" name="buttonReIndex">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ReIndex</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonTest">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="LabelConsoleOutput">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Console Output</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidgetInput">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>300</width>
     <height>160</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="console">
    <attribute name="title">
     <string>Console</string>
    </attribute>
    <widget class="QPushButton" name="consoleCopy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>115</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="consoleClear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>115</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="consoleSwitch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>115</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Deactivate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="consolOutputSelect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>101</y>
       <width>155</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>All Information</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sampling Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Re-Indexing Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Debug</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="seperationLine">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>280</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelOutputSelection">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>77</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Selection:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vlConsole">
      <item>
       <widget class="QCheckBox" name="consoleLineNumbers">
        <property name="text">
         <string>Line Numbers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="consoleTimeStamp">
        <property name="text">
         <string>Time Stamp</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="visualization">
    <attribute name="title">
     <string>Visualization</string>
    </attribute>
    <widget class="QComboBox" name="visualDrawingMethod">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>5</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>point cloud</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wireframe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>flat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>smooth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>texture</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>9</y>
       <width>115</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Drawing Method:</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>280</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutSampling">
      <item>
       <widget class="QCheckBox" name="visualInvertNormals">
        <property name="text">
         <string>invert normals</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="visualVertexWeights">
        <property name="text">
         <string>vertex weights</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="visualSampledVertices">
        <property name="text">
         <string>sampled vertices</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>139</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutRemeshing">
      <item>
       <widget class="QCheckBox" name="visualControlPoints">
        <property name="text">
         <string>control points</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="visualRemeshedRegions">
        <property name="text">
         <string>remeshed regions</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="visualDecimatedMesh">
        <property name="text">
         <string>decimated mesh</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="interation">
    <attribute name="title">
     <string>Interaction</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTextEdit" name="ConsoleOutput">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>260</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;... console loaded:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="boxMeshInfo">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>0</y>
     <width>145</width>
     <height>175</height>
    </rect>
   </property>
   <property name="title">
    <string>Information</string>
   </property>
   <widget class="QLabel" name="labelNumberVertices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>130</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mesh Vertices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNumberVerticesValue">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSampledVertices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sampled Vertices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSampledVerticesValue">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSampledVerticesRatio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
